Tools/samples.h

```diff
@@ -74,8 +74,8 @@ namespace AnaSamples
         }
 
         const T& null() const {return nullT_;}
-    protected:
         std::map<std::string, T> sampleSet_;
+    protected:
```

Comment:
I would rather keep this variable protected, what functionality does this allow?